ts/defs.bzl

```diff
@@ -346,12 +344,37 @@ def ts_project(
         else:
             fail("transpiler attribute should be a rule/macro or a skylib partial. Got " + type(transpiler))
 
+        # Default target produced by the macro gives the js and map outs, with the transitive dependencies.
+        js_library(
+            name = name,
+            # Include the tsc target in srcs to pick-up both the direct & transitive declaration outputs so
```

Comment:
Why srcs and not deps? I guess I'm not super clear on when we use which one, it's a little confusing because we collect from both

Reply:
I see it's preexisting, so just wondering

Reply:
I think because both js and dts files are the primary output of this target, we don't want to consider them as transitive deps.